addons/website_slides/tests/test_channel_utils.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.website_slides.tests import common as slides_common
+from odoo.tests.common import users
+
+
+class TestSlidesManagement(slides_common.SlidesCase):
+
+    @users('user_publisher')
+    def test_get_categorized_slides(self):
+        base_domain = []
+        order = self.env['slide.slide']._order_by_strategy['sequence']
+        categorized_slides = self.channel._get_categorized_slides(base_domain, order)
```

Comment:
categorized_slides = self.channel._get_categorized_slides([], order)
and remove base_domain